src/VirtoCommerce.ReturnModule.Web/Scripts/blades/items-list.tpl.html

```diff
@@ -11,29 +11,30 @@
                 <table class="table">
                     <thead>
                         <tr>
-                            <th class="table-col __product-control">
+                            <th class="table-col __product-control" ng-show="!blade.editMode">
```

Comment:
I'm not sure about using ng-show statement. Usually, we are using ng-if to prevent content from blinking while the page is loading.

Reply:
It's still ng-show

Reply:
Replacement the remaining ng-show corrupts the layout.